src/node_debug_options.cc

```diff
@@ -93,15 +93,15 @@ bool DebugOptions::ParseOption(const std::string& option) {
   }
 
   // --debug and --inspect are mutually exclusive
-  if (option_name == "--debug") {
+  /*if (option_name == "--debug") {
```

Comment:
`--debug` is the primary way to activate the legacy debugger interface. Aliasing it to `--inspect` instead would make it impossible to start that interface. That would break tools which still depend on it.

I think we should keep `--debug` associated with the legacy debugger as long as possible, i.e. till we update to V8 5.8+.